docs/debug/2_config_file_structure.rst

```diff
@@ -0,0 +1,240 @@
+..
+    Copyright (c) 2022-2025, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+
+    See LICENSE for license information.
+
+Config File Structure
+====================
+
+To enable debug features, create a ``config.yaml`` file to specify the desired behavior, such as determining which GEMMs (General Matrix Multiply operations) should run in higher precision rather than FP8 and defining which statistics to log. Below, we outline how to structure the ``config.yaml`` file.
+
+General Format
+-------------
+
+A config file can have one or more sections, each containing settings for specific layers and features:
+
+.. code-block:: yaml
+
+    section_name_1:
+      enabled: ...
+      layers:
+        # Specify layers here...
```

Comment:
Could you add example here, since from the getting started it seems you want a list e.g [layer1, layer2] rather than 
layer1:
layer2:
for example

Reply:
Ok, scratch that - I see you do that in the next section.